graylog2-web-interface/src/pages/ViewEventDefinitionPage.tsx

```diff
@@ -81,11 +83,13 @@ const ViewEventDefinitionPage = () => {
       <PageHeader title={`View "${eventDefinition.title}" Event Definition`}
                   actions={(
                     <ButtonToolbar>
+                      {isMutable && (
```

Comment:
@kingzacko1 What do you think about, in the case where the definition is not mutable, adding some text here explaining that they can't edit "System/Illuminate" event definitions and to make a copy if they want to customize?

Just anticipating people not understanding what's happening and thinking this is a bug or something.

Otherwise this looks solid and tests successfully!

Reply:
The `SystemNotificationEventEntityScope` actually is mutable so it would solely be Illuminate events definitions without this button right now. Also, it might look a little out of place if we were to try and shove some text where buttons are supposed to be. What do you think instead about adding some boilerplate to the description of each event definition that gets shipped with Illuminate? Something like `Event definitions shipped with Illuminate cannot be directly modified. Please duplicate this event definition and then make modifications to the copy.` at the end of each description. @ryan-carroll-graylog

Reply:
Ah yeah good point, yeah I think having a note on the Illuminate event definition works if that's if it's not hard to do.
Were you thinking updating the actual bundles or adding it somewhere in the Illuminate install process?

One other thing I was thinking was swapping the "Edit" button for a "Duplicate" button (Maybe with a tooltip explaining why?), but not sure how feasible that is.

In any case, not really sure how deep we want to go since this is just meant to be a temporary guardrail. Maybe @rich-graylog has a preference?

Reply:
I did think about potentially adding a `Duplicate` button in the toolbar just generally for event definitions, I think that would make sense as opposed to making the user click back to the list of event definitions to duplicate it.

Reply:
@ryan-carroll-graylog latest changes have a `Duplicate Event Definition` button for all event definition view pages (minus `System notification events`). Duplicating from this page opens the newly created event definition for editing to limit a couple clicks.